src/Includes/IncludedExists.php

```
@@ -2,17 +2,22 @@
 
 class IncludedExists implements IncludeInterface
 {
-    public function __invoke(Builder $query, string $exists)
+    public function __construct(
+        protected ?Closure $constraint = null,
+    ) {}
+
+    public function __invoke(Builder $query, string $exists): void
     {
-        $exists = Str::before($exists, config('query-builder.exists_suffix', 'Exists'));
+        $exists = Str::before($exists, config('query-builder.suffixes.exists', 'Exists'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Str' in method '__invoke'. (undefined)

(StaticAccess)